app/controllers/annotations_controller.rb

```diff
@@ -144,7 +152,11 @@ def update_annotation
     @content = params[:annotation_text][:content]
     @id = params[:annotation_text][:id]
     @submission_file_id = params[:submission_file_id]
```

Comment:
The current implementation of editing an existing annotation (from the annotations tab), only passes the `annotation_text_id` rather than the `annotation_id`. I don't think I am able to obtain the `@annotation.submission_file` from the `annotation_text_id`, unless I am able to look up the `annotation_id` from the `annotation_text_id`. With that being said, another way I may be able to get rid of `submission_file` altogether in `update_annotation` and `destroy` is if I have the original implementation for editing an annotation pass in `annotation_id` instead of `annotation_text_id`. Should I proceed to make this change?